Bigquery collector (#13599)
netdata/netdata: collectors/python.d.plugin/bigquery/README.md

<!--
title: "BigQuery"
custom_edit_url: https://github.com/netdata/netdata/edit/master/collectors/python.d.plugin/bigquery/README.md
-->

# BigQuery Netdata Collector

A python based collector leveraging [`pandas-gbq`](https://pandas-gbq.readthedocs.io/en/latest/) to run a query 
against [BigQuery](https://cloud.google.com/bigquery) and save results into a chart in Netdata.

## Requirements

> This collector depends on some Python packages that can usually be installed via `pip`.

Reviewer: Nowadays we still have have `pip` and `pip3` on hosts, my initial understanding, and probably I am wrong, this collector needs `python 2`. @Ferroin am I missing something with these `pip` versions?

Author: Looks like `pandas-gbq` is only python 3. https://pypi.org/project/pandas-gbq/

So i'll need to add something maybe to make this clearer.

Reviewer: I think so, only to be sure that users won't have the confusion I had.

> This collector depends on some Python packages that can usually be installed via `pip`.

Author: ```suggestion
This collector depends on some Python (Python 3 only) packages that can usually be installed via `pip` or `pip3`.
```

Reviewer: Perfect, with this I think the PR will be ready to be merged. :) 
What do you think @Ferroin ?

```bash

> sudo pip install google-auth pandas-gbq

Reviewer: please, add a `$` in front bash

> sudo pip install google-auth pandas-gbq

Author: ```suggestion
$ sudo pip install google-auth pandas-gbq
```

```

## Credentials

The collector expects to read the credentials for a 
[service account](https://cloud.google.com/iam/docs/service-accounts) from a file defined in the collector 

> configuration (see `credentials` configuration parameter). If this is not defined then auth will fallback 

Reviewer 2: ```suggestion
configuration (see `credentials` configuration parameter). If this is not defined, then auth will fall back 
```

to the `pandas-gbq` [approach](https://pandas-gbq.readthedocs.io/en/latest/howto/authentication.html). 

## Configuration

> Below is an example configuration to just query some random data from BigQuery (two times) and plot each 

Reviewer 2: ```suggestion
Below is an example configuration to query some random data from BigQuery (two times) and plot each 
```
